src/lib/utils/theme.utils.ts

```diff
@@ -0,0 +1,56 @@
+import { Theme } from "../types/theme";
+import { enumFromStringExists } from "./enum.utils";
+
+const isNode = (): boolean =>
+  typeof process !== "undefined" &&
+  process.versions != null &&
+  process.versions.node != null;
+
+export const THEME_ATTRIBUTE = "theme";
+export const LOCALSTORAGE_THEME_KEY = "nnsTheme";
+
+export const initTheme = (): Theme => {
+  // Jest NodeJS environment has no document
+  if (isNode()) {
+    return Theme.DARK;
+  }
+
+  const theme: string | null =
+    document.documentElement.getAttribute(THEME_ATTRIBUTE);
+
+  const initialTheme: Theme = enumFromStringExists({
+    obj: Theme as unknown as Theme,
```

Comment:
I guess `as unknown as Theme` is not needed anymore. At least it doesn't complains locally...

Reply:
Beautiful